app/src/ui/clone-repository/clone-repository.tsx

```diff
@@ -372,4 +379,22 @@ export class CloneRepository extends React.Component<
 
     setDefaultDir(Path.resolve(path, '..'))
   }
+
+  private onFocus = () => {
```

Comment:
Could we rename this to be more clear about the fact that this method is called when the window receives focus, not the component itself. In `dialog.tsx` we called it `onWindowFocus` which I think is a good name.